src/docs/get-started/codelab.md

```diff
@@ -243,8 +243,8 @@ packages, on [pub.dev][].
          return MaterialApp(
            title: 'Welcome to Flutter',
            home: Scaffold(
-    @@ -16,7 +18,7 @@
-               title: Text('Welcome to Flutter'),
+               appBar: AppBar(
```

Comment:
I think this line might be prefixed with 2 extra spaces. Can you make sure this is formatted correctly?

Reply:
Yep